### ISSUE TITLE
AnkiDroid 2.15.6 missing from "overview"

### ISSUE THREAD

**Issue by choralbari** [ankidroid/Anki-Android#9951]
Since updating to AnkiDroid 2.15.16, AnkiDroid never shows up in the overview (that is, the scrollable list of active apps accessed through the square button on Android's nav bar). In order to switch from another app back to AnkiDroid, something I do frequently when creating cards, I have to go back to the home screen and re-select the app. Tapping the square button in the other app also won't switch back to AnkiDroid. I don't recall this being an issue before.

I am running Android 6.0 on a Moto e2.

**Comment by mikehardy** [comment]
Hi there! That sounds frustrating, but unfortunately I cannot reproduce. I just verified on my device (Android 11) that it is there in the app switcher. I also verified an API23 emulator (that's Android 6) and it's there.

This may be device or operating system specific. Someone with a Moto e2 will have to try it or perhaps there are some system settings for your device or for your launcher that you have set.

There are other reports of this and it looks like an OS or launcher issue. People are saying to reboot phone or force quit the affected app https://search.brave.com/search?q=app+not+showing+in+recent+apps+android

This is certainly nothing that we are doing in the app and unfortunately there is nothing we could even change here to affect it, which means that it is not actionable. We only keep items open in the issues list that are actionable in order to avoid clutter, so I'm going to close this but hopefully what information I've found can point you in the right direction

Cheers
